### ISSUE TITLE
How to install?

### ISSUE THREAD

**Issue by LecramRed** [martinbosma/dbus-homewizard#1]
Can you give an instruction how to install this on VenusOS?

**Comment by martinbosma** [comment]
This project is far from finished and hasn't been maintained for 2 years.
It is a personal project, so not officially supported by Victron.

If you want to install this, have a look at the wiki of VenusOS: 
https://github.com/victronenergy/venus/wiki/howto-add-a-driver-to-Venus#installing-a-driver-that-doesnt-depend-on-a-serial-port

You will be needing root access to your GX as well: https://www.victronenergy.com/live/ccgx:root_access

**Comment by martinbosma** [comment]
You also need the `Zeroconf` python module, which is not installed by default.

**Comment by LecramRed** [comment]
Thank you Martin. Is there a change that the Homewizard P1 and energy meters will get supported by Victron? Many people in NL have these products and if the data is reliable, it will save a grid meter and additional PV meters. I have the P1 meter at home and an EM24, the PV groups have their HW energy meters and Shelly for Victron. It would be great if the HW will work with Victron as well.

**Comment by martinbosma** [comment]
No, not that I'm aware of. Perhaps I will look into this project again for my personal needs, but not any time soon. Besides, most P1 output is too slow; for proper ESS control you would need an update rate < 1Hz.

**Comment by MacStephane** [comment]
Hi Martinbosma,
Any idea how to install Zeroconf? When running your script this appears in the log: 
" ModuleNotFoundError: No module named 'zeroconf'"
I already tried: "pip install zeroconf" this gave me following error: "ModuleNotFoundError: No module named 'colorsys'"
Thanks in advance and kind regards.

**Comment by martinbosma** [comment]
@MacStephane If you try to install on VenusOS, try the following:
```
opkg update
opkg install python3-pip
pip3 install zeroconf==0.38.1
```

That zeroconf version was released around the time I made dbus-homewizard. I have not tested this with the actual script, just tested if zeroconf would install on VenusOS v3.20~7. 

If the rootfs is read-only, have a look at [this](https://www.victronenergy.com/live/ccgx:root_access#partitions_read-only_rootfs_and_available_disk_space) guide.
